accounts/client.go

```diff
@@ -125,27 +167,23 @@ func (c *Client) RetrievePayoutSchedule(entityId string) (*PayoutSchedule, error
 	return &response, nil
 }
 
-func (c *Client) UpdatePayoutSchedule(
-	entityId string,
-	currency common.Currency,
-	updateSchedule CurrencySchedule,
-) (*common.IdResponse, error) {
+func (c *Client) QueryPaymentInstruments(entityId string, query PaymentInstrumentsQuery) (*PaymentInstrumentQueryResponse, error) {
 	auth, err := c.configuration.Credentials.GetAuthorization(configuration.OAuth)
 	if err != nil {
 		return nil, err
 	}
 
-	request := map[common.Currency]CurrencySchedule{
-		currency: updateSchedule,
+	url, err := common.BuildQueryPath(common.BuildPath(accountsPath, entitiesPath, entityId, PaymentInstrumentsPath), query)
+
+	if err != nil {
+		return nil, err
 	}
 
-	var response common.IdResponse
-	err = c.apiClient.Put(
-		common.BuildPath(accountsPath, entitiesPath, entityId, payoutSchedulesPath),
+	var response PaymentInstrumentQueryResponse
+	err = c.apiClient.Get(
```

Comment:
This method call can be done in one line now because it is shorter than before

Reply:
👍